integration_tests/components/datePicker.ts

```diff
@@ -0,0 +1,38 @@
+import { PageElement } from '../pages/page'
+
+export default class DatePickerComponent {
```

Comment:
Moving selector methods for Date Picker to new class so they can be re-used between pages that have the component.